### PR TITLE
Oracle v6 documentation

### PR DESCRIPTION
**Pull request by sudarshan12s**

### CODE REVIEW

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md [hunk 1]
```diff
@@ -199,6 +199,31 @@ For running integration test:
 SEQ_ACCOUNT=myAccount SEQ_USER=myUser SEQ_PW=myPassword SEQ_ROLE=myRole SEQ_DB=myDatabaseName SEQ_SCHEMA=mySchema SEQ_WH=myWareHouse npm run test-integration-snowflake
 ```
 
+### Oracle Database
+
+The underlying connector library used by Sequelize for Oracle is the [node-oracledb](https://www.npmjs.com/package/oracledb) package.  
+See [Releases](/releases#oracle-support-table) to see which versions of Oracle Database & node-oracledb are supported.
+
+node-oracledb needs IC[Oracle Instant Client](https://www.oracle.com/database/technologies/instant-client/downloads.html) to work. You can use node-oracledb [quick start](https://oracle.github.io/node-oracledb/INSTALL.html#quickstart) link for installations.
```
Comment (cjbj) [inline versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:207]: Don't use "IC" - just say Oracle Instant Client.  

In the 2nd sentence add "the" before "node-oracledb"

**Review by cjbj — CHANGES_REQUESTED** [review]

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md (hunk 1 above)
Comment (sudarshan12s) [reply versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md]: incorporated the changes.

File: versioned_docs/version-6.x.x/getting-started.md [hunk 2]
```diff
@@ -44,7 +45,7 @@ const sequelize = new Sequelize({
 // Option 3: Passing parameters separately (other dialects)
 const sequelize = new Sequelize('database', 'username', 'password', {
   host: 'localhost',
-  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' */
+  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' | 'oracle' */
```
Comment (sdepold) [inline versioned_docs/version-6.x.x/getting-started.md:48]: Oh. I think we are actually lacking some others here. Would you mind expanding it to

'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql' | 'db2' | 'snowflake' | 'oracle'

**Review by sdepold — APPROVED** [review]

**Review by fzn0x — DISMISSED** [review]

File: src/pages/releases.mdx [hunk 3]
```diff
@@ -161,6 +161,19 @@ support for Db2 for IBM i is limited as it is not handled by the core team.
...
+Oracle Database requires the use of the [node-oracledb] npm package.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#oracle).
+
+| Sequelize   | [Oracle Database]   | [node-oracledb] |
+|-------------|---------------------|-----------------|
+| 7 (alpha)   |      upcoming       |    upcoming     |
+| 6 (current) |      >= 18.4        |    ^5.4.0       |
+
+[Oracle Database]: https://www.oracle.com/database/technologies/oracle-database-software-downloads.html
+[oracledb]: https://www.npmjs.com/package/oracledb
```
Comment (cjbj) [inline src/pages/releases.mdx:175]: This should be `[node-oracledb]`

File: versioned_docs/version-6.x.x/getting-started.md (hunk 2 above)
Comment (sudarshan12s) [reply versioned_docs/version-6.x.x/getting-started.md]: changes done.

File: src/pages/releases.mdx (hunk 3 above)
Comment (sudarshan12s) [reply src/pages/releases.mdx]: corrected.

**Review by sdepold — APPROVED** [review]

**Pull request merged**